ui/src/components/dashboard/components/selector/Selector.vue

```diff
@@ -12,7 +12,7 @@
                     type="primary"
                     :icon="Plus"
                     tag="router-link"
-                    :to="{name: 'dashboards/create', query}"
```

Comment:
I have a feeling that this was used for something, but can't figure out what, do you have ideas?

If unused anymore, then all good.